apiml-security-common/src/test/java/org/zowe/apiml/security/common/content/BearerContentFilterTest.java

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
Not using `given_when_then` format

Reply:
I was referring to the other filters similar tests and forgot to adopt this structure :( I've fixed it, thanks for the catch